.eslintrc.js

```diff
@@ -0,0 +1,77 @@
+"use strict";
+
+module.exports = {
+    root: true,
+    extends: ["eslint"],
+    parserOptions: {
+        ecmaVersion: 2019
```

Comment:
I'd ideally like to start using ES2015+ features, but I do think it's important we stick to features available in Espree so that we can dogfood it in our site. Does limiting ourselves to features available in Espree (so, features that have reached stage 4 and are supported by the parser) sound like a good policy?

Reply:
I think we should in the README (and maybe also create a CONTRIBUTING.md document)